server/src/test/java/io/spine/examples/chatspn/server/chat/ChatTest.java

```diff
@@ -96,6 +102,21 @@ void personalChatCreation() {
         context().assertState(command.getId(), expectedState);
     }
 
+    @Test
+    @DisplayName("create a `ChatPreview` projection and update the `UserChats` projection" +
+            "after personal chat creation")
```

Comment:
Please review the concatenation of display names here and below. I think, space symbols are missing.